website/src/components/Messages/MessageTableEntry.tsx

```diff
@@ -93,7 +95,7 @@ export function MessageTableEntry({ message, enabled, highlight }: MessageTableE
           style={{ float: "right", position: "relative", right: "-0.3em", bottom: "-0em", marginLeft: "1em" }}
           onClick={(e) => e.stopPropagation()}
         >
-          {Object.entries(emojiState.emojis).map(([emoji, count]) => (
+          {Object.entries(emojiState?.emojis || {}).map(([emoji, count]) => (
```

Comment:
How could values in `emojiState` be undefined? they areis assigned values on init? maybe it is the `useEffect`? if so, we should provide the default values there.

Reply:
That's a good deduction.  Done.